erpnext/portal/doctype/homepage/test_homepage.py

```diff
@@ -1,19 +1,126 @@
 # -*- coding: utf-8 -*-
-# Copyright (c) 2019, Frappe Technologies Pvt. Ltd. and Contributors
+# Copyright (c) 2020, Frappe Technologies Pvt. Ltd. and Contributors
 # See license.txt
 from __future__ import unicode_literals
 
 import frappe
 import unittest
 from frappe.utils import set_request
+from bs4 import BeautifulSoup
 from frappe.website.render import render
 
 class TestHomepage(unittest.TestCase):
+	def setup_test_data(self):
```

Comment:
Name this method "setUp" and it will get executed automatically before every test case. (Check the unittest docs for details.)